package.json

```diff
@@ -53,6 +53,7 @@
     "@types/bluebird": "^3.5.26",
     "@types/node": "^10.12.27",
     "@types/validator": "^10.9.0",
+    "big-integer": "^1.6.42",
```

Comment:
Shim for BigInt, which is new in Node 10. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt.